app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+    @classmethod
+    def __str__(cls) -> str:
```

Comment:
The `__str__` method is typically an instance method, not a class method. Consider removing the `@classmethod` decorator and adjusting the method to return a string representation of an instance, rather than the class variable `alive`.